book/src/voluntary-exit.md

```diff
@@ -25,7 +25,7 @@ In order to initiate an exit, users can use the `lighthouse account validator ex
 
 - The `--keystore` flag is used to specify the path to the EIP-2335 voting keystore for the validator.
 
-- The `--beacon-node` flag is used to specify a beacon chain HTTP endpoint that confirms to the [Eth2.0 Standard API](https://ethereum.github.io/eth2.0-APIs/) specifications. That beacon node will be used to validate and propagate the voluntary exit. The default value for this flag is `http://localhost:5052`.
+- The `--beacon-nodes` flag is used to specify a beacon chain HTTP endpoint that confirms to the [Eth2.0 Standard API](https://ethereum.github.io/eth2.0-APIs/) specifications. That beacon node will be used to validate and propagate the voluntary exit. The default value for this flag is `http://localhost:5052`.
```

Comment:
```suggestion
- The `--beacon-nodes` flag is used to specify one ore more beacon chain HTTP endpoints that confirm to the [Eth2.0 Standard API](https://ethereum.github.io/eth2.0-APIs/) specifications. Those beacon nodes will be used to validate and propagate the voluntary exit. The default value for this flag is `http://localhost:5052`.
```

Reply:
I thought about that but didn't make the change thinking that this is not a general description of the `--beacon-nodes` flag, rather it describes the flag in the context of initiating a voluntary exit where I would imagine specifying more than one beacon node is quite rare. So I opted for simplicity.

I will of course still make the change if you think it's better.